kotlin/src/main/com/looker/rtl/ApiSettings.kt

```diff
@@ -64,31 +86,22 @@ open class ApiSettings(contents: String, var section: String = "") : Configurati
     override var headers: Map<String, String> = mapOf()
 
     init {
-        val config = apiConfig(contents)
-        section = if (!section.isBlank()) section else config.keys.first()
+        val settings = rawReadConfig();
 
-        if (config[section].isNullOrEmpty()) {
-            throw Error("No section named '$section' was found")
+        // Only replace the current values if new values are provided
+        settings["base_url"].let { value ->
+            baseUrl = unQuote(value ?: baseUrl)
         }
 
-        config[section]?.let { settings ->
-
-            // Only replace the current values if new values are provided
-            settings["base_url"].let { value ->
-                baseUrl = unQuote(value ?: baseUrl)
-            }
-
-            settings["api_version"].let { value ->
-                apiVersion = unQuote(value ?: apiVersion)
-            }
-
-            settings["verify_ssl"].let { value ->
-                verifySSL = asBoolean(value) ?: verifySSL
-            }
-            settings["timeout"].let { value ->
-                timeout = if (value !== null) value.toInt() else timeout
-            }
+        settings["api_version"].let { value ->
+            apiVersion = unQuote(value ?: apiVersion)
+        }
```

Comment:
I think we don't need this any more at all, but that's not necessary to fix in this PR